modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,79 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as a task to that namespace.
+
+Use {pipelines-shortname} to run pods in the user namespace.
+
+.Prerequisites
+
+* Pass the annotations that CRIO expects.
+
+.Procedure
+
+. Use the `oc edit clustertask buildah` command to edit the buildah cluster task.
+
+. Add the annotations for all users.
+
+. Export the cluster task as a task to the namepsace.
+
+. Use the `tkn create` feature to automate the creation of a cluster task.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc new-project test
+----
++
+[source,terminal,subs="attributes+"]
+----
+$ tkn task create --from=buildah
```

Comment:
It looks like `tkn task create` isn't documented. This is the only mention of `tkn` and `create` is here: https://docs.openshift.com/container-platform/4.10/cli_reference/tkn_cli/op-tkn-reference.html#resource-create
Please create a separate Jira bug report to document it.

Reply:
Keeping this open to remind me about opening a Jira for this.

Reply:
@HarshCasper, I created https://issues.redhat.com/browse/RHDEVDOCS-4184. Please close this comment as resolved.